polkadot/node/core/provisioner/src/lib.rs

```diff
@@ -214,12 +214,9 @@ async fn handle_active_leaves_update(
 		if per_session.get(&session_index).is_none() {
 			let prospective_parachains_mode =
 				prospective_parachains_mode(sender, leaf.hash).await?;
-			let elastic_scaling_mvp = request_node_features(leaf.hash, session_index, sender)
-				.await?
-				.unwrap_or(NodeFeatures::EMPTY)
-				.get(FeatureIndex::ElasticScalingMVP as usize)
-				.map(|b| *b)
-				.unwrap_or(false);
+
+			// elastic scaling always on
+			let elastic_scaling_mvp = true;
```

Comment:
same as before, let's restructure/simplify the code

Reply:
Done